### DISCUSSION TITLE
How many tabs can a tab tab before being tabbed out?

### DISCUSSION THREAD

**Discussion by Urethramancer** [community/community#8022] [Copilot Conversations]
When Copilot gets in that one line at a time state, it also requires me to keep pressing tab for each step of indentation before it actually completes a line. This gets more tedious the deeper the nesting for a function goes. A five-liner with 3 levels of indentation requires me to press tab 13 times, plus return after each line. It's getting a bit much.

**Comment by jansorg** [comment]
@Urethramancer Which IDE or editor are you using?

**Reply by Urethramancer** [reply]
VSCode.
